cmd/postgres.go

```diff
@@ -679,6 +679,11 @@ func runListPostgresDatabases(cmdCtx *cmdctx.CmdContext) error {
 		return fmt.Errorf("get app: %w", err)
 	}
 
+	// check if app is a postgres app
+	if app.ImageDetails.Repository != "flyio/postgres" && app.ImageDetails.Repository != "flyio/postgres-standalone" {
```

Comment:
We probably shouldn't duplicate this code, especially with hardcoded values. Can this be moved to a function?